README.md

```diff
@@ -8,10 +8,17 @@ Please read the [contribution guidelines](CONTRIBUTING.md).
 
 ## Setting up Development Environment
 
-Floworky uses [`node-foreman`](https://github.com/strongloop/node-foreman) to manage process startup.  This requires the use of a `.env` file to specify the environment variables required by floworky (note that this is an example that includes values that will need to be set for your own environment):
+Floworky uses [`node-foreman`](https://github.com/strongloop/node-foreman) to  
```

Comment:
Please remove the new lines at the end of the line - markdown files are the one exception to the 80 character limit (since they are pretty printed when they're rendered).  :P